src/Button/Button.tsx

```diff
@@ -123,7 +123,7 @@ export class Button extends React.PureComponent<Button.Props> {
   static defaultProps: ButtonDefaultProps = {
     textOverflow: TextOverflow,
     buttonState: 'ready',
-    size: 'medium',
+    size: 'small',
```

Comment:
is this breaking change necessary?

Reply:
🤷‍♂️ The idea of this PR is to have consistent sizes by default.